test/core/address_utils/parse_address_test.cc

```diff
@@ -80,6 +83,31 @@ static void test_grpc_parse_unix_abstract(const char* uri_text,
 
 #endif  // GRPC_HAVE_UNIX_SOCKET
 
+#ifdef GRPC_HAVE_VSOCK
+
+static void test_grpc_parse_vsock(const char* uri_text, uint32_t cid, uint32_t port) {
+  grpc_core::ExecCtx exec_ctx;
+  absl::StatusOr<grpc_core::URI> uri = grpc_core::URI::Parse(uri_text);
+  if (!uri.ok()) {
+    gpr_log(GPR_ERROR, "%s", uri.status().ToString().c_str());
+    ASSERT_TRUE(uri.ok());
+  }
+  grpc_resolved_address addr;
+
+  ASSERT_TRUE(grpc_parse_uri(*uri, &addr));
+  struct sockaddr_vm* addr_vm =
+      reinterpret_cast<struct sockaddr_vm*>(addr.addr);
+  ASSERT_EQ(AF_VSOCK, addr_vm->svm_family);
+  ASSERT_EQ(grpc_ntohl(addr_vm->svm_cid), cid);
+  ASSERT_EQ(addr_vm->svm_port, port);
+}
+
+#else  // GRPC_HAVE_VSOCK
+
+static void test_grpc_parse_vsock(const char* uri_text, uint32_t cid, uint32_t port) {}
```

Comment:
I believe these arguments will be identified as unused at compile time on platforms that don't have VSOCK. Suggest `test_grpc_parse_vsock(const char* /* uri_text */, ...)`

Reply:
done